src/app/shared/components/tag-list/style.scss

```diff
@@ -17,3 +17,9 @@
 :host {
   width: 100%;
 }
+
+.mat-standard-chip {
+  .mat-chip-remove {
+    background: black;
```

Comment:
Does it work with dark and light themes?

Reply:
Anyways it should be exported to theme.scss.

Reply:
Good pointer, you are right it should be exported to theme.